core/src/main/java/tachyon/web/UiFileInfo.java

```diff
@@ -108,6 +118,19 @@ public UiFileInfo(LocalFileInfo fileInfo) {
     mIsDirectory = fileInfo.mIsDirectory;
     mIsPinned = false;
     mFileLocations = new ArrayList<String>();
+    for (int i = 0; i < StorageLevelAlias.SIZE; i ++) {
+      mBlocksOnTier.add(new ArrayList<UiBlockInfo>());
+    }
```

Comment:
Yes. I added this lines just to make sure the getBlocksOnTier() method is always valid for any UiFileInfo object.